web/src/settings_users.js

```diff
@@ -579,7 +583,7 @@ function handle_filter_change($tbody, section) {
     // can't use that, because we're also filtering on Role with our
     // custom predicate.
     $tbody
-        .closest(".settings-section")
+        .closest(".user-settings-section")
```

Comment:
While this change doesn't change anything behaviour-wise, it's better to use `user-settings-section` after each tab in the users page started using `user-settings-section` instead of `settings-section`